testsuite/tests/tool-caml-tex/redirections.reference

```diff
@@ -21,6 +21,10 @@
 \caml\camlinput\?1 + \<2.\> ;;
 \endcamlinput\camlerror\:Error: This expression has type float but an expression was expected of type
 \:         int
+\:File "/home/jules/ocaml/testsuite/tests/tool-caml-tex/redirections.ml", line 1, characters 2-3:
```

Comment:
There is no way that the test can pass with this location information here. You should modify the tested to avoid this suberror for now, and I will have a look at caml-tex handling of suberrors.

Reply:
This is what I got with `make promote`. I updated the test.